test/build/runner.js

```diff
@@ -0,0 +1,84 @@
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs').promises;
+const {readdirSync, readFileSync, statSync, existsSync} = require('fs');
+const execa = require('execa');
+const dircompare = require('dir-compare');
+
+function stripWhitespace(stdout) {
+  return stdout.replace(/((\s+$)|((\\r\\n)|(\\n)))/gm, '');
+}
+function stripRev(code) {
+  return code.replace(/\?rev=\w+/gm, '?rev=XXXXXXXXXX');
+}
+function stripChunkHash(stdout) {
+  return stdout.replace(/([\w\-]+\-)[a-z0-9]{8}(\.js)/g, '$1XXXXXXXX$2');
+}
+
+beforeAll(() => {
+  // Needed so that ora (spinner) doesn't use platform-specific characters
+  process.env = Object.assign(process.env, {CI: '1'});
+});
+
+for (const testName of readdirSync(__dirname)) {
+  if (testName === 'node_modules' || testName.includes('.')) {
+    continue;
+  }
+
+  test(testName, async () => {
+    let cwd = path.join(__dirname, testName);
+
+    // add exception for yarn-workspaces test
+    if (testName === 'yarn-workspaces') {
+      cwd = path.join(cwd, 'packages', 'snowpack');
+    }
```

Comment:
This is a noisy PR, but this is really the main file to pay attention to: `runner.js`. It’s basically a copy of the other test, with a few minor tweaks:

- Doesn’t compare expected output
- Doesn’t compare `snowpack.lock.json`